governance/meeting-host-playbook.md

```diff
@@ -0,0 +1,85 @@
+# TAG Environmental Sustainability Meeting Host Playbook
+
+Meetings are held to coordinate and discuss TAG ENV activities. The TAG ENV leads host these meetings. The following playbook can be used for reference to structure the meeting, you don't need to stick step by step (word by word), it should give guidance how a meeting usually is structured.
+
+## Requirements to host the meeting
+
+* Install Zoom and check your settings (especially audio and video).
+* Get access to the [CNCF TAG YouTube Channel](https://www.youtube.com/@CNCFEnvTAG) to start the livestream.
+
+## Pre-Meeting
+* Add a new blank agenda entry to the meeting notes by copying the template and editing the details accordingly.
+* Announce a day in advance in Slack to add agenda items to meeting notes.
+* Announce the meeting about an hour before in the `#tag-env-sustainability` Slack channel.
+
+```
+Hi folks! The CNCF TAG ENV bi-weekly meeting takes place 1 hour from now
+Links:
+* Agenda & Minutes: https://bit.ly/cncf-tag-env-meeting-notes
+* Zoom Link: https://zoom.us/my/cncftagenvsustainability, passcode `77777`
+
+If anyone has anything they’d like to discuss, please add to it to the agenda, thanks!
+```
+
+## During the Meeting
+
+### Preparation
+
+* Slack reminder (thread) *"The meeting start now!"*
+* [Join](https://zoom.us/my/cncftagenvsustainability) the meeting
+* Turn on your camera
```

Comment:
Maybe don't make this a requirement.  Sometimes, people may be in a place that camera on is not appropriate or their camera may just be a finicky creature.

Reply:
Turn on **your** camera

there is a couple of lines below

> * *"If you rather not get recorded, you can now turn off your video"*

Reply:
resolved?